torchvision/prototype/transforms/__init__.py

```diff
@@ -10,5 +10,4 @@
 from ._geometry import HorizontalFlip, Resize, CenterCrop, RandomResizedCrop, FiveCrop, TenCrop, BatchMultiCrop
 from ._meta import ConvertBoundingBoxFormat, ConvertImageDtype, ConvertImageColorSpace
 from ._misc import Identity, Normalize, ToDtype, Lambda
-from ._presets import CocoEval, ImageNetEval, VocEval, Kinect400Eval, RaftEval
```

Comment:
Removing them from the `transforms` namespace. These classes are considered private and users shouldn't be initiatlizing them directly. TorchVision models return them as callables and the users can use them but we reserve their right to change them on the future as long as the preset transforms of the pre-trained models remain correct.